holoviews/tests/operation/test_datashader.py

```diff
@@ -1169,6 +1171,24 @@ def test_rasterize_image_expand_default(self):
         output = img.data["z"].to_numpy()
         assert np.isnan(output).any()
 
+    def test_rasterize_apply_when_instance_with_line_width(self):
+        df = pd.DataFrame(
+            np.random.multivariate_normal(
+            (0, 0), [[0.1, 0.1], [0.1, 1.0]], (500000,))
+        )
+        df.columns = ["a", "b"]
+
+        curve = Curve(df, kdims=["a"], vdims=["b"])
+        custom_rasterize = rasterize.instance(line_width=2)
```

Comment:
Maybe add a comment about what is so special about `line_width` e.g. it is not a parameter.